src/lib/FlightTasks/tasks/FlightTaskManualStabilized.hpp

```diff
@@ -49,26 +49,30 @@ class FlightTaskManualStabilized : public FlightTaskManual
 
 	virtual ~FlightTaskManualStabilized() = default;
 
-	bool activate() override;
+	void initialiseOutputs(ControlSetpoint &setpoint);
```

Comment:
To me both names are good and self-explanatory. However, we should think about British initialise vs American initialize

Reply:
I vote for `activate`, that's why I named it like this in the first place. It's the callback that a task has which gets called when the task is activated...